build/deploy/schema-manager.libsonnet

```diff
@@ -0,0 +1,51 @@
+local base = import 'base.libsonnet';
+local volumes = import 'volumes.libsonnet';
+local defaultdb_schema = import "db-schemas/defaultdb.libsonnet";
+
+local rid_schema_vol = {
+  name: 'db-rid-schema',
+  configMap: {
+    defaultMode: 420,
+    name: 'db-rid-schema',
+  },
+};
+local rid_schema_mount = {
+  name: 'db-rid-schema',
+  readOnly: false,
+  mountPath: '/db-schemas/defaultdb',
+};
+
+{
+  all(metadata): {
+    rid_schema: base.ConfigMap(metadata, 'db-rid-schema') {
+      data: defaultdb_schema.data
+    },
+    RIDSchemaManager: if metadata.cockroach.shouldInit then base.Job(metadata, 'rid-schema-manager') {
```

Comment:
For this config it is a pre requisite because it is meant for the first run that bootstraps the database schema. I did this because we are going to remove the bootstrapping from GRPC backend